src/addTodo.js

```diff
@@ -0,0 +1,11 @@
+export default (todos, todo) => {
+  if (todo) {
+    const index = todos.length + 1;
+    todos.push({ description: todo, completed: false, index });
+    localStorage.setItem('todos', JSON.stringify(todos));
+  } else {
+    /* eslint-disable */
+    alert('Cannot add blank todo!');
```

Comment:
Please remove alert and confirmation from the project.  Don't use window.alert() or window.confirm() in the project. Use custom modules instead.